ui/src/api/wfegraph.js

```diff
@@ -17,28 +18,45 @@ class Workflow2Graph {
 
     const forks = [];
     const tasks = wfe.tasks || [];
+    this.executedTasks = {};
     tasks.forEach(tt=>{
-      if(tt.taskType === 'FORK'){
+      this.executedTasks[tt.referenceTaskName] = {
+        status: tt.status,
+        input: tt.inputData,
+        output: tt.outputData,
+        taskType: tt.taskType,
+        reasonForIncompletion:
+        tt.reasonForIncompletion,
+        task: tt
+      };
+
+      if(tt.type === 'FORK'){
         let wfts = [];
         let forkedTasks = tt.inputData && tt.inputData.forkedTasks || [];
         forkedTasks.forEach(ft =>{
-          wfts.push({name: ft, referenceTaskName: ft, type: 'SIMPLE'});
+          wfts.push({name: ft, taskReferenceName: ft, type: 'SIMPLE'});
         });
-        forks[tt.referenceTaskName] = wfts;
+        forks[tt.taskReferenceName] = wfts;
       }
-    });
+    }), this;
```

Comment:
I think the parentheses is misplaced.  Should it be moved to after the 'this'?

Reply:
Good catch @mstier-nflx 

What's interesting is that I just triple-checked, and this actually still builds and compiles and works. I am changing the parenthesis as you suggested......just not sure why. Maybe there's something we depend on or webpack itself that compiles this correctly? Really weird that it works this way :shrug: